src/System.Net.Requests/src/System/Net/_TLSstream.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net
+{
+    internal class TlsStream : NetworkStream
```

Comment:
Thanks for explaining. I've noticed @Priya91 already added this at `src/Common/src/System/Net/TlsStream.cs`.
Would that work here too?